app/components/autocomplete/autocomplete.js

```diff
@@ -261,39 +262,41 @@ export default class Autocomplete extends PureComponent {
 const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
     return {
         base: {
-            left: 0,
-            overflow: 'hidden',
+            left: 8,
```

Comment:
but it may break the modifiers in the search screen, so be mindful of that and look for an alternative. Maybe is breaking as there is something rendering (invisible) when it shouldn't be